docs/target-systems/mongodb-target-system.md

```diff
@@ -65,22 +65,11 @@ These dependencies must be provided at target system creation time with **no glo
 | `MongoClient` | `mongoClient` | MongoDB connection client - **required** for both target system configuration and change execution |
 | `String` | `databaseName` | Target database name - **required** to identify which database changes will affect |
 
-### Optional Configuration (.withXXX() methods)
```

Comment:
Why are we removing this configuration options? In springdata TS are included. Lets discuss about it.